index.html

```diff
@@ -21,158 +21,174 @@
 
   <body>
     <div class="header">
+      <a href="#">
+        <img
+          src="images/bikes-for-refugees_logo.jpg"
+          alt="Logo for Bikes for Refugees Scotland"height="60px"width="06px";
+          class="header__logo"
+        />
+      </a>
+
+      <div>
+        <ul class="navigation__list">
+          <li class="navigation__item">
+            <a class="navigation__link" href="#">Home</a>
+          </li>
+          <li class="navigation__item">
+            <a class="navigation__link" href="#">About Us</a>
+          </li>
+          <li class="navigation__item">
+            <a class="navigation__link" href="#">News & Events</a>
+          </li>
+          <li class="navigation__item">
+            <a class="navigation__link" href="#">Contact Us</a>
+          </li>
+        </ul>
+      </div>
 
-        <a href="#">
-          <img
-            src="images/bikes-for-refugees_logo.jpg"
-            alt="Logo for Bikes for Refugees Scotland"
-            class="header__logo"
-          />
-        </a>
-
-        <div>
-          <ul class="navigation__list">
-            <li class="navigation__item">
-              <a class="navigation__link" href="#">Home</a>
-            </li>
-            <li class="navigation__item">
-              <a class="navigation__link" href="#">About Us</a>
-            </li>
-            <li class="navigation__item">
-              <a class="navigation__link" href="#">News & Events</a>
-            </li>
-            <li class="navigation__item">
-              <a class="navigation__link" href="#">Contact Us</a>
-            </li>
-          </ul>
-        </div>
-
-        <button>Donate Now</button>
-
+      <button>
+        <a href="https://codeyourfuture.io/donate/"> Donate Now </a>
+      </button>
     </div>
 
     <div class="content">
-        <div class="main">
-          <div class="alert">
-            You've helped us give <span class="text-highlight">72</span> bikes
-            to refugees and asylum seekers so far. <strong>Thank you!</strong>
-          </div>
+      <div class="main">
+        <div class="alert">
+          You've helped us give <span class="text-highlight">72</span> bikes to
+          refugees and asylum seekers so far. <strong>Thank you!</strong>
+        </div>
 
-          <div class="hero">
-            <h1>Bikes for Refugees</h1>
-            <p>
-              Providing donated bikes and accessories to refugees and asylum
-              seekers in Scotland.
-            </p>
-            <button>Donate a bike today</button>
-            <button>Volunteer</button>
+        <div class="hero">
+          <h1>Bikes for Refugees</h1>
+          <p>
+            Providing donated bikes and accessories to refugees and asylum
+            seekers in Scotland.
+          </p>
+          <div class="button-div">
```

Comment:
You don't need to add this div in. As mentioned the divs are not semantic, whereas the button tag is.